content/hardware/08.edu/solution-and-kits/alvik/tutorials/user-manual/user-manual.md

```diff
@@ -454,3 +1609,19 @@ Now you can open the Arduino Lab for MicroPython, connect Alvik and open the exa
 7. Test `bender.py`

```

Comment:
let's get rid of the Bender example please XD, I am pretty sure we can make other nice stuff with the display instead of this one

Reply:
Removed